app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+class ArticlesController < ApplicationController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable '@article' [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->